simulator/infra/ports/ports.h

```diff
@@ -439,6 +439,7 @@ decltype(auto) make_read_port(Args... args)
 }
 
 static constexpr const Latency PORT_LATENCY = 1_Lt;
+static constexpr const Latency PORT_LONG_LATENCY = 2_Lt;
```

Comment:
That is too small value to see any impact of cache. Please set it to 30.

Reply:
Pavel Igorevich, Is it possible to set latency to 30? If we have miss, we'll push bubble into pipeline, but 10 bubbles in a row is a deadlock( there we will have bubbles in 30 cycles).

Reply:
Increase deadlock threshold to 100.
In real HW stalls of thousand cycles are possible.